lmcache/storage_backend/serde/__init__.py

```diff
@@ -12,6 +13,8 @@ def CreateSerde(serde_type: str, config: LMCacheEngineConfig, metadata: LMCacheE
         s, d = SafeSerializer(), SafeDeserializer()
     elif serde_type == "cachegen":
         s, d = CacheGenSerializer(config, metadata), CacheGenDeserializer(config, metadata)
+    elif serde_type == "fast":
```

Comment:
I would like to directly replace the current "torch serializer/deserializer". But depends on your preference.